src/components/button/__snapshots__/button.spec.js.snap

```diff
@@ -0,0 +1,1109 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Theme object is repeated multiple times in these snapshots. It could be mapped to a string, when using `toJson` method of the `enzyme-to-json` library.